examples/webpack.config.js

```diff
@@ -0,0 +1,81 @@
+const path = require('path');
+const fs = require('fs');
+const webpack = require('webpack');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const BundleAnalyzerPlugin =
+  require('webpack-bundle-analyzer').BundleAnalyzerPlugin;
+const { CleanWebpackPlugin } = require('clean-webpack-plugin');
+
+const outputDirectory = 'webpack-output';
+
+// Dynamically find all the examples. Note: This only works for one level deep
+// and a root file name of index.jsx.
+const entries = fs
+  .readdirSync(path.resolve(__dirname), { withFileTypes: true })
+  .filter((dirent) => dirent.isDirectory())
+  .filter((dirent) => dirent.name !== outputDirectory)
+  .reduce(
+    (compiledList, dirent) => ({
+      ...compiledList,
+      [dirent.name]: path.resolve(__dirname, `${dirent.name}/index.jsx`),
+    }),
+    {}
+  );
+entries.root = path.resolve(__dirname, 'index.jsx');
+
+const config = {
+  mode: 'development',
+  entry: entries,
+  output: {
+    path: path.resolve(__dirname, outputDirectory),
+    filename: '[name]/bundle.js',
+  },
+  module: {
+    rules: [
+      {
+        test: /\.(ts|tsx|js|jsx)$/,
+        use: 'babel-loader',
+        exclude: /node_modules/,
+      },
+      {
+        test: /\.ts(x)?$/,
+        loader: 'ts-loader',
+        exclude: /node_modules/,
+        options: {
+          configFile: path.resolve(__dirname, '../tsconfig.json'),
```

Comment:
I think we might be able to get those errors to go away if we use a different `tsconfig.json` which [extends](https://www.typescriptlang.org/tsconfig#extends) from the base. In this extended config, we could put `"test/"` into the [exclude](https://www.typescriptlang.org/tsconfig#exclude) array. 

Alternatively, we may be able to exclude `test/` right here on the same line where we're excluding `node_modules/`

Reply:
Ooh, both good ideas. I'll try excluding it first before making a new `tsconfig.json` which extends the base.

Reply:
Excluding it in the loader didn't work, which makes sense. Because webpack isn't actually loading these modules. The TypeScript compiler is because of the `tsconfig.json`. I'll try creating a new one and extending the existing.

Reply:
That did the trick. :raised_hands: